gulpfile.js

```diff
@@ -169,3 +174,31 @@ gulp.task('test/node', ['compile'], function(cb) {
   jrunner.addSpecFiles(specFiles);
   jrunner.execute();
 });
+
+gulp.task('test/nativescript', ['compile'], function(cb) {
```

Comment:
Please also update the `package.json` file so that native scripts can be run using `npm run test-nativescript` command.